Python/sqlmlutils/packagemanagement/servermethods.py

```diff
@@ -21,45 +18,8 @@ def get_server_info():
     else:
         from pip import pep425tags
     return {
-        "impl_version_info": pep425tags.get_impl_version_info(),
-        "abbr_impl": pep425tags.get_abbr_impl(),
-        "abi_tag": pep425tags.get_abi_tag(),
-        "platform": pep425tags.get_platform()
-    }
-
-
-def check_package_install_success(sql_package_name: str) -> bool:
-    return package_exists_in_scope(sql_package_name)

```

Comment:
Is all of this dead code and no longer needed ? Thats why we remove it ?

Reply:
I actually don't think it was ever used. 

There may have been a time where this code was used but it's not just in a spees script elsewhere. I don't know when it was "deprecated" but I noticed there are no references to this code so I removed it.

---
In reply to: [501256877](https://github.com/microsoft/sqlmlutils/pull/82#discussion_r501256877) [](ancestors = 501256877)